test/int.jl

```diff
@@ -241,6 +241,10 @@ end
     @test typeof(widen(Int64(-3))) == Int128
     @test typeof(widen(Int128(-3))) == BigInt
 
+    @test widen(UInt16(65535)) === UInt32(65535)
```

Comment:
Am I missing something or these (and more) methods are already tested in https://github.com/JuliaLang/julia/blob/afa04068636ca6bc3edce6c110e9bb79818d1a7a/test/int.jl#L220-L230?